docs/tutorials/connecting-to-the-internet-via-usb.md

```diff
@@ -135,6 +135,99 @@ If you are using an older version, please upgrade.
 
         {% include /util/screenshot.html source="/images/brickman/wired-status-online-connect-automatically-selected.png" %}
 
+
+*   {: tab="linux_command_line"}{% include /style/icon.html type="info" %}
+    These instructions were written using Debian look-alikes for those wanting to know what happens under the hood.
+    The commands need to be executed by root.
+    {:class="alert alert-info"}
+
+    1.  Booting with a standard ev3dev image, lsusb will show
+
+        `Bus 002 Device 008: ID 0694:0005 Lego Group`
+        
+        I'll be using the EV3 as a USB Gadget, as follows.
+
+    2.  When you've copied the ev3dev's image to the SD-card, mount the second partition (EV3DEV_ROOTFS).
```

Comment:
Why are we doing this? ev3dev already sets up a gadget driver by default. Does it not work for you?